recipe/conda_build_config.yaml

```diff
@@ -1,3 +1,6 @@
 MACOSX_SDK_VERSION:        # [osx and x86_64]
-  - "10.12"                # [osx and x86_64]
+  - "10.13"                # [osx and x86_64]
 
+# upstream requires at least 1.78 as of v2022.07
+boost_cpp:
+  - 1.79.0
```

Comment:
Let's go back to 1.78 now that you figured out the osx-errors?

Reply:
Yes, agreed, though I'd like to update the global pin and avoid setting it here altogether.